README.md

```diff
@@ -278,3 +278,33 @@ The relevant pages can be checked to adopt a similar design.
 2. Find the color section you wish the change (it is commented)
 3. Change the variable here only.
 4. Do not change the code below the variables unless you wish to add or remove a color instead of changing a color.
+
+### How to add new icons?
+
```

Comment:
Add a sentence here to tell the reader what this section is about, what we hope to accomplish after following the steps.